air/src/air.rs

```diff
@@ -47,10 +47,14 @@ pub trait Air<AB: AirBuilder>: BaseAir<AB::F> {
 /// as well cases where the constraints are evaluated on an evaluation trace and combined using randomness.
 pub trait AirBuilder: Sized {
     /// Underlying field type.
-    type F: Field;
+    type F: Field + Into<Self::I>;
```

Comment:
Minor question of if it's better to enforce `From` or `Into`. Doesn't really matter but `From` might be cleaner particularly if you do the `Algebra` suggestion below.
```suggestion
    type F: Field;
```

Reply:
Let's keep `From` then